### ISSUE TITLE
Livecharts 2 Avalonia 11 Preview Several Bugs

### ISSUE THREAD

**Issue by Jon-Murray** [beto-rodriguez/LiveCharts2#676]
Hey Beto,

The new Avalonia preview stuff looks really cool, thanks so much for getting this up before it is even general release! Unfortunately, it seems there are a few bugs. I have a few samples from an application using old livecharts/avalonia 10, and the same app (no code modifications) using avalonia 11 - bugs appear in both avalonia 11 preview 1 and avalonia 11 preview 2.

### First Bug:

**It is impossible to set chart background colours/they are always transparent.** This is not immediately obvious on the sample application as it uses ```TransparencyLevelHint="Blur"``` on the topmost window, which seems to be the only way to change the control (other than manually setting the background colour/opacity in MotionCanvas.axaml.cs, but that doesn't really work properly either). If you remove ```TransparencyLevelHint="Blur"``` then it will always be fully transparent, no matter how you wrap it, if you set the background colour or opacity of the control/chart/any parents/children, etc.
The below screenshot is of the sample application, with the background set to black/TransparencyLevelHint and no further changes (you can see the code/Visual Studio in the background):

![Demo new](https://user-images.githubusercontent.com/2168795/194857866-0ea16dfd-5840-45c8-829d-5c7cb11d16d8.png)

### Second Bug:

**Tooltip icons are incorrectly displayed when using a formatter.** I am using a custom formatter which is really basic. nb. that this bug might trigger without a custom formatter, but I honestly just noticed it when I was using a custom one. However, I am using the ```TooltipBackground``` property to set a brush. In the old version it worked fine, in the new version it has a box around the series:

Old version: 
![tooltip_original](https://user-images.githubusercontent.com/2168795/194858349-0113536a-ef41-4f53-9d64-b91a7ef90ccc.png)

New version:
![tooltip1](https://user-images.githubusercontent.com/2168795/194858428-dceb31ae-2449-4fca-b89b-581db30f48c2.png)

You can also see this in the screenshot of the first bug (in the legend on the right hand side) 

### Third Bug:
**Nested pie charts with visible borders/improperly overlapping.** I have 2 pie charts, the first inside the second. I have a "SeriesOuter" and a "SeriesInner", where SeriesInner would always total the value of SeriesOuter. For example, if I had a SeriesOuter entry with the value of 50, I'd have n SeriesInner values of 25/25 or 10/10/20/5/5, (any number of values, as long as they add up to a total 50). The values I used to achieve this are below:

```
       SeriesOuter.Add(new PieSeries<somemodel>()
                            {
                                Values = new List<somemodel>() { val },
                                InnerRadius = 90,
                                HoverPushout = 0,
                                MaxOuterRadius = 1,
                                Fill = new SolidColorPaint(outercolour),
                                TooltipLabelFormatter = RenderTooltipOuter,
                                Mapping = MapOuter
                            });

...

   (inside a loop)

   SeriesInner.Add(new PieSeries<somemodel>()
                                {
                                    Values = new List<somemodel>() { somevaluefirst},
                                    InnerRadius = 70,
                                    HoverPushout = 0,
                                    MaxOuterRadius = 1,
                                    Fill = new SolidColorPaint(innercolour1),
                                    TooltipLabelFormatter = RenderTooltipInner,
                                    Mapping = MapInner
                                });

                               (end loop)
```

On the old version it looks like this:

![pie original](https://user-images.githubusercontent.com/2168795/194859729-6cf6efc8-a55c-4b33-bf0b-bb1bcb670305.png)

On the new version it looks like this:
(nb. the square right in the very middle was my crappy photoshop skills, trying to crop irrelevant data, please ignore it!)

![pie 1](https://user-images.githubusercontent.com/2168795/194859789-03474d26-57ee-435a-b71d-06e14f0ce145.png)

(note, on the original version, on the outer pie chart all cursor events would fire/you could mouseover all regions, this is broken on the new one... the square regions i've highlighted on the outer pie do not fire the mouseover event for the chart... most likely because it is "blocked" by the rendering of the inner pie.)

Also note that I have tried changing the rendering order. Usually, the inner is drawn first, then the outer. Drawing the outer first, then drawing the inner inside it, will completely block the inner one, it will not render/nest at all.

I've also tried setting the z-index property of the outer/inner appropriately, and no dice. 

Sorry for such a long-winded message, but if there's anything I can help with, or you need more code to repro anything please let me know. Thanks again for the great work, the library is so cool.

**Comment by HobbesKo** [comment]
Hi Jon-Murray

Regarding your First Bug with the transparent background, I only encountered this issue when updating LiveCharts to 2.0.0-beta.501-11.0.0-preview1. Reverting to ...beta.500... resolved the problem.

**Comment by Jon-Murray** [comment]
@HobbesKo You're a life saver! I can confirm that using the initial commit at https://github.com/beto-rodriguez/LiveCharts2/tree/13d9866b679df42ac6c6c9c99e475791ca5eaf0d fixes all 3 of the issues i've listed here. Thanks!

**Comment by beto-rodriguez** [comment]
Thanks for the hints, then I know the mistake is there!

**Comment by beto-rodriguez** [comment]
Thanks for the report this is fixed since `2.0.0-beta.502-11.0.0-preview1` version.
